components/MonitoringCommentFrame.vue

```diff
@@ -0,0 +1,61 @@
+<template>
+  <div>
+    <div class="MonitoringCommentFrame" :style="{ color: `${colors[level]}` }">
+      <p>{{ $t(comment) }}</p>
+    </div>
+    <div
+      class="MonitoringCommentFrameAfter"
+      :style="{ color: `${colors[level]}`, background: `${colors[level]}` }"
+    />
+  </div>
+</template>
+
+<script>
+import Vue from 'vue'
+
+export default Vue.extend({
+  props: {
+    level: {
+      type: Number,
+      required: true,
+      default: 0
+    },
+    comment: {
+      type: String,
+      required: true,
+      default: ''
+    }
+  },
+  data() {
+    return {
+      colors: ['#96cd5f', '#fffc4e', '#fdbe40', '#fa1629']
+    }
+  }
+})
+</script>
+
+<style lang="scss" scoped>
+.MonitoringCommentFrame {
+  border: solid 1px;
+  text-align: center;
+  margin-bottom: 10px;
+  > p {
+    margin-top: auto;
+    margin-bottom: auto;
+    position: relative;
+    font-style: normal;
+    font-weight: bold;
+    color: $gray-2 !important;
```

Comment:
そうすると、 `!important` が要らなくなるはず。
```suggestion
    color: $gray-2;
```

Reply:
この `!important` は
https://github.com/tokyo-metropolitan-gov/covid19/pull/5029/files#diff-a654a20e4c54ef10e895ef3f8f322b22R115
の`$gray-4`が上書きされて色が薄くなることに対するものですね。

`!important`を使うのがあまりよろしくないのはわかっていますが、対策がわからず...

Reply:
ああ、なるほどですね。本来なら `<p>感染状況</p>` にclassをつけるのがスマートだと思います。まあ、できればでいいですよ！

Reply:
今からやるのはちょっと厳しいので、必要であればお願いしたいです...

Reply:
OKです！私のほうでやっておきますね、ここまでご対応いただいたことに感謝です！